spacemesh/v1/admin_types.proto

```diff
@@ -15,3 +17,84 @@ message RecoverRequest {
   string uri = 1;
   uint32 restore_layer = 2;
 }
+
+message Event {
+  // time when event occured.
+  google.protobuf.Timestamp timestamp = 1;
+  // the reason of the failure may be complex to pinpoint.
+  // for the first version we want to highlight that failure occured and defer user to logs. 
+  bool failure = 2;
+  string help = 3;
+  oneof details {
+    EventBeacon beacon = 4;
+    EventInitStart init_start = 5;
+    EventInitComplete init_complete = 6;
+    EventPostStart post_start = 7;
+    EventPostComplete post_complete = 8;
+    EventPoetWait poet_wait = 9;
+    EventPoetWaitChallenge poet_wait_challenge = 10;
```

Comment:
If I understand correctly, this is supposed to notify that nipostBuilder is waiting for poet round to finish, right? If that's the case, consider:
```suggestion
    EventPoetWaitForProof poet_wait_for_proof = 10;
```

Reply:
i can change if you prefer. challenge was in the context of the post, because what we receive from poet is used as a challenge in the post